docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -318,6 +318,13 @@ In addition, you also should pay particular attention to MySQL's encoding. Altho
     Read https://stackoverflow.com/questions/9192027/invalid-default-value-for-create-date-timestamp-field for how to disable it.
     See `SQL Mode - NO_ZERO_DATE <https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_no_zero_date>`__ for more information.
 
+.. warning::
+  In Airflow 2.9.0 we have changed xcom table column value type from blob to longblob. This will allow you to store relatively big data in Xcom but process can take a significant amount of time if you have a lot of large data stored in Xcom.
```

Comment:
This should go in a newsfragment instead -  [docs here](https://github.com/apache/airflow/blob/83d62cae695ebea0e537c7bfa8e9cc65de05ec09/contributing-docs/16_contribution_workflow.rst#step-4-prepare-pr).